source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,268 @@
+    - ``PostID``: Unique identifier of the post .
+    - ``NumChecklists``: Number of checklists. It always equals 1.
+    - ``TotalChecklistItems``: Number of checklist items this incident starts with. It always equals 0."
+    "Incident finished.", "- Any user sends the ``/incident end`` command. 
+    - Any user clicks on the ``End Incident`` button through the incident details view, in the RHS.", "
+    - ``ID``: Unique identifier of the incident.
+    - ``IsActive``: Boolean  value indicating if the incident is active. It always equals ``false``.
+    - ``CommanderUserID``: Unique identifier of the commander of the incident. It equals the identifier of the user that created the incident.
+    - ``UserID``: Unique identifier of user that ended the incident.
+    - ``TeamID``: Unique identifier of the team where the incident channel is created.
+    - ``CreatedAt``: Timestamp of the incident creation.
+    - ``ChannelIDs``: A list containing a single element, the channel created along with the incident.
+    - ``PostID``: Unique identifier of the post .
+    - ``NumChecklists``: Number of checklists. It always equals 1.
+    - ``TotalChecklistItems``: Number of checklist items this incident starts with. It always equals 0."
+    "Checklist item created", "- Any user creates a new checklist item through the incident details view, in the RHS.", "
+    - ``IncidentID``: Unique identifier of the incident where the item was created.
+    - ``UserID``: Unique identifier of the user that created the item."
+    "Checklist item removed.", "- Any user deletes a checklist item through the incident details view, in the RHS.", "
+    - ``IncidentID``: Unique identifier of the incident where the item was.
+    - ``UserID``: Unique identifier of the user that removed the item."
+    "Checklist item renamed.", "- Any user edit the contents of a checklist item through the incident details view, in the RHS.", "
+    - ``IncidentID``: Unique identifier of the incident where the item was.
+    - ``UserID``: Unique identifier of the user that removed the item."
+    "Checklist item moved.", "- Any user moves the position of a checklist item in the list through the incident details view, in the RHS.", "
+    - ``IncidentID``: Unique identifier of the incident where the item is.
+    - ``UserID``: Unique identifier of the user that edited the item."
+    "Unchecked checklist item checked.", "- Any user checks an unchecked checklist item through the incident details view, in the RHS.", "
+    - ``IncidentID``: Unique identifier of the incident where the item is.
+    - ``UserID``: Unique identifier of the user that checked the item."
+    "Checked checklist item unchecked.", "- Any user unchecks a checked checklist item through the incident details view, in the RHS.", "
+    - ``IncidentID``: Unique identifier of the incident where the item is.
+    - ``UserID``: Unique identifier of the user that checked the item."
```

Comment:
@agarciamontoro, is this meant to be "checked" or "unchecked"?

Reply:
Oopsie, it should be unchecked! Good catch, because I worded this quite confusingly!